src/main/js/components/content/generator/Schedules.jsx

```diff
@@ -137,7 +137,7 @@ class Schedules extends React.Component {
         </div>
         <div className="carousel">
           <div className="carousel-inner">
-            { showTextTable && <CourseTable courses={schedule} colored header /> }
+            { showTextTable && <CourseTable courses={schedule} gradeMap={gradeMap} sortByGPA={sortByGPA} colored header /> }
```

Comment:
Since `gradeMap` and `sortByGPA` are only being used to pass down to `CourseTable`, it make sense to make `CourseTable` container instead under `containers/generator`.  It should basically look like `Schedules.js`. Don't forget to import the container in `Schedules.jsx` once you've made it.

Reply:
Done.